### ISSUE TITLE
Invalid value of type 'builtins.str' received for the 'customdata' property of scatter

### ISSUE THREAD

**Issue by anitagraser** [ghtmtt/DataPlotly#84]
Using QGIS 3.3, I get the following error trying to create scatter plots or bar plots. (Histograms work fine.) The data is from NaturalEarth (ne_110m_populated_places dataset) and the fields I'm trying to use are numeric.

```
2018-10-27T14:57:15     WARNING    Traceback (most recent call last):
              File "C:/Users/anita/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\DataPlotly\data_plotly_dialog.py", line 1000, in createPlot
              self.plotProperties()
              File "C:/Users/anita/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\DataPlotly\data_plotly_dialog.py", line 973, in plotProperties
              self.plotobject.buildTrace()
              File "C:/Users/anita/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\DataPlotly\data_plotly_plot.py", line 189, in buildTrace
              opacity=self.plot_properties['opacity']
              File "C:\OSGEO4~1\apps\Python37\lib\site-packages\plotly\graph_objs\_scatter.py", line 2285, in __init__
              self['customdata'] = customdata if customdata is not None else _v
              File "C:\OSGEO4~1\apps\Python37\lib\site-packages\plotly\basedatatypes.py", line 2775, in __setitem__
              self._set_prop(prop, value)
              File "C:\OSGEO4~1\apps\Python37\lib\site-packages\plotly\basedatatypes.py", line 3011, in _set_prop
              raise err
              File "C:\OSGEO4~1\apps\Python37\lib\site-packages\plotly\basedatatypes.py", line 3006, in _set_prop
              val = validator.validate_coerce(val)
              File "C:\OSGEO4~1\apps\Python37\lib\site-packages\_plotly_utils\basevalidators.py", line 341, in validate_coerce
              self.raise_invalid_val(v)
              File "C:\OSGEO4~1\apps\Python37\lib\site-packages\_plotly_utils\basevalidators.py", line 244, in raise_invalid_val
              valid_clr_desc=self.description()))
             ValueError: 
              Invalid value of type 'builtins.str' received for the 'customdata' property of scatter
              Received value: 'POP1975'
             
              The 'customdata' property is an array that may be specified as a tuple,
              list, numpy array, or pandas Series
```

**Comment by anitagraser** [comment]
It's 3.3.0 (using OSGeo4W installer)

**Comment by ghtmtt** [comment]
in the meanwhile you can download the zip from https://github.com/ghtmtt/DataPlotly/tree/plotly3 branch and install it in QGIS. It is not ready (yet)  to be released as an updated version of the plugin

**Comment by ghtmtt** [comment]
Should be fixed in 244d0d458c2aad15a5ab3b476b852e51e2fa4e74

Released version 2.0 for QGIS 3.4. Closing the issue, please reopen if necessary.
